### PR TITLE
CA: DAP-3150, DAP-3151, DAP-3161

### PR DESCRIPTION
**Pull request by Ni-2**
#### What is the purpose of this pull request? (put an "X" next to an item)

[x] New feature

#### What changes did you make? (Give an overview)

DAP-3150: WalletConnect(ETH) support in Connected Accounts

DAP-3151: show CA list for Rthereum wallets

DAP-3161: add dropodown for selecting wallet to show CA list

### CODE REVIEW

**Review by alsakhaev — CHANGES_REQUESTED** [review]

File: src/contentscript/overlay/root/components/CAUserButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export const CAUserButton = ({
+  user,
+  handleOpenPopup,
+  maxLength = 32,
+  colour = 'white',
+}: {
+  user?: IConnectedAccountUser
+  handleOpenPopup?: (account: IConnectedAccountUser) => Promise<void>
+  maxLength?: number
+  colour?: string
```
[the remark this answers is not in the document]
Comment (Ni-2) [reply src/contentscript/overlay/root/components/CAUserButton/index.tsx:16]: https://www.google.com/search?q=color+or+colour&oq=color+or+colour&aqs=chrome..69i57j0i512l9.5462j0j7&sourceid=chrome&ie=UTF-8

[the remark this answers is not in the document]
Comment (Ni-2) [reply src/contentscript/overlay/root/components/CAUserButton/index.tsx]: fixed

File: src/contentscript/overlay/root/components/CAUserButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import styles from './CAUserButton.module.scss'
+
+export const CAUserButton = ({
+  user,
+  handleOpenPopup,
+  maxLength = 32,
+  colour = 'white',
+}: {
+  user?: IConnectedAccountUser
+  handleOpenPopup?: (account: IConnectedAccountUser) => Promise<void>
```
[the remark this answers is not in the document]
Comment (Ni-2) [reply src/contentscript/overlay/root/components/CAUserButton/index.tsx]: fixed

**Pull request merged**